jobs/JGLOBAL_PREP_OCEAN_OBS

```diff
@@ -14,13 +14,13 @@ YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COMOUT_OBS:COM_OBS_TMPL
 # Begin JOB SPECIFIC work
 ##############################################
 
-# Add prep_marine_obs.py to PYTHONPATH
-export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush:${PYTHONPATH}
+# Add prep_ocean_obs.py to PYTHONPATH
+export PYTHONPATH=${HOMEgfs}/ush:${PYTHONPATH}
```

Comment:
```suggestion
```
Can we move this to the rocoto job?
The j-job is not the place to define/update/add `PYTHONPATH`

Reply:
done, and retested